css/styles.css

```diff
@@ -0,0 +1,97 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ OPTIONAL ] Kindly ensure that the font family, font size, line height, and font weight of the text in your body stick to the Figma design you chose as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). You can check out the Figma design specification for the exact properties used👍🏽 
- [ OPTIONAL ]  I suggest importing this font family I show you below  so that you can apply it in your all  body :+1: :grinning: :grinning: :grinning: 

```
@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");

 font-family: "Poppins", sans-serif;

```